udata/tests/test_search.py

```diff
@@ -122,6 +123,11 @@ def get_query(facet_search):
 class SearchQueryTest(SearchTestMixin, TestCase):
     maxDiff = None
 
+    def assert_dict_equal(self, d1, d2, *args, **kwargs):
```

Comment:
I think that we should stick to the `TestCase` case with `assertDictEqual` for consistency. By the way, do we need the `*args, **kwargs` parameters?

What about putting this helper at the `TestCase` level for it to be reuse elsewhere?